postcss.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  plugins: [
+    require('autoprefixer')({
+      browsers: ['> 5%']
```

Comment:
cool, I meant to leave a comment asking what value would be good for this setting. > 1% it is!